app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(i["name"], i["age"]) for i in people]
+
+    for i in people:
```

Comment:
Avoid 1-letter loop variable name. It should be descriptive. Rename it everywhere.